### PR TITLE
fix: Tabs inkBar in StrictMode

### LINKED ISSUE #39494: Tabs not showing underline below tab title
### Reproduction link

[https://github.com/hbatalhaStch/antd-tabs-no-underline-bug](https://github.com/hbatalhaStch/antd-tabs-no-underline-bug)

### Steps to reproduce

just run `npm start`

### What is expected?

Blue underline below the tab title

### What is actually happening?

Only showing the tab title, no underline
image

| Environment | Info |
| --- | --- |
| antd | 5.0.6 |
| React | 18.2.0 |
| System | Windows 10 Pro; Version: 21H2; OS Build: 19044.2251 |
| Browser | Microsoft Edge Version 108.0.1462.46 (Official build) (64-bit) |

---
Only happens with Typescript, running the same project in Javascript works fine

### LINKED ISSUE #39631: inkBar of Tabs is not working with React.StrictMode
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/peaceful-sinoussi-2cig5l)

### Steps to reproduce

1. Create a React 18 project
2. Install Antd 5.0.7, import Tabs and reset.css
3. Add Tabs (basic)
4. Use the Tabs wrapped in <React.StrictMode><Tabs></React.StrictMode>
5. InkBar will not show
6. Remove the <React.StrictMode>
7. InkBar will show normally

### What is expected?

InkBar is expected to show using React StrictMode

### What is actually happening?

Width of InkBar become 0 (zero) in its css.

| Environment | Info |
| --- | --- |
| antd | 5.0.7 |
| React | 18.2.0 |
| System | Windows 10 |
| Browser | Chrome 109.0.5414.25 |

---
There was a similar problem in the past, issue 35367

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

fix #39631
fix #39494

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix Tabs inkBar not show in StrictMode.      |
| 🇨🇳 Chinese |    修复 Tabs inkBar 在 StrictMode 下不展示的问题。    |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: components/tabs/__tests__/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -263,6 +267,7 @@ exports[`Tabs tabBarGutter should work 2`] = `
       >
         <div
           class="ant-tabs-tab ant-tabs-tab-active"
+          data-node-key="null"
```
Comment (afc163) [inline components/tabs/__tests__/__snapshots__/index.test.tsx.snap:270]: null 需要渲染出来么？

Comment (zombieJ) [reply components/tabs/__tests__/__snapshots__/index.test.tsx.snap:270]: 像是个 bug，我瞅瞅

Comment (zombieJ) [reply components/tabs/__tests__/__snapshots__/index.test.tsx.snap:270]: > 像是个 bug，我瞅瞅

Update：发现是测试用例写的比较随便，没给 key。另外 `data-node-key` 是 `querySelector` 需要用，总是有的。

**Comment by FeNoMeNa** [comment]
On the latest versions of Chrome the problem is fixed, but it seems that there is a problem for the older ones. I've tested with Chrome 45 (Ant 5.12) and the inkBar is missing.

**Pull request merged**